.changeset/six-bobcats-impress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@fuel-ts/versions": patch
+"@internal/forc": patch
```

Comment:
There shouldn't be a need for this package in the changesets anymore. We should ignore it.